API.md

```diff
@@ -0,0 +1,308 @@
+Status: 404 Not Found
+{
+  "message": "Not existing book"
+}
+```
+
+## Edit book
+```
+PUT /books/:id
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| :---: |:---:| ---|
+| title | String | Book title |
+| author | String | Author |
+| description | String | Description |
+| image | String | Image URL |
+
+**Example**
+```
+{
+  "title": "Sprint : How to Solve Big Problems and Test New Ideas in Just 5 Days",
+  "author": "Jake Knapp y John Zeratsky",
+  "description": "The sprint is a five-day process for answering critical business questions through design, prototyping, and testing ideas with customers.",
+  "image": "https://images-na.ssl-images-amazon.com/images/I/512V%2BzNxQ9L._SX327_BO1,204,203,200_.jpg"
+}
+```
+
+**Response**
+```
+Status 200
+  
+{
+  "id": "33",
+  "title": "Sprint : How to Solve Big Problems and Test New Ideas in Just 5 Days",
+  "author": "Jake Knapp y John Zeratsky",
+  "description": "The sprint is a five-day process for answering critical business questions through design, prototyping, and testing ideas with customers.",
+  "image": "https://images-na.ssl-images-amazon.com/images/I/512V%2BzNxQ9L._SX327_BO1,204,203,200_.jpg"
+}
+```
+
+## Get Books
+```
+GET /books
+```
+
+**Response**
+```
+Status 200
+  
+[
+  {
+    "id": "33",
+    "title": "Sprint : How to Solve Big Problems and Test New Ideas in Just 5 Days",
+    "author": "Jake Knapp y John Zeratsky",
+    "description": "The sprint is a five-day process for answering critical business questions through design, prototyping, and testing ideas with customers.",
+    "image": "https://images-na.ssl-images-amazon.com/images/I/512V%2BzNxQ9L._SX327_BO1,204,203,200_.jpg"
+  },
+  ...
+]
+```
+
+# Meals
+## Get Next Menu
+```
+GET /menus/next
+```
+
+**Response**
+```
+Status 200
+  
+{
+  "date": "2017-5-1T14:00:00Z",
+  "firstCourses": [
+    {
+      "name": "Salad",
+      "description": "Tomatoes, lettice and bacon.",
+      "image": "https://www.google.es/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwjknLawgs3TAhULVxQKHfbcAvMQjRwIBw&url=http%3A%2F%2Fjetspizza.com%2Fmenu%2Fsalads&psig=AFQjCNE447cnpJfGRhiwiN5Hys_XLrgpmQ&ust=1493670502918204"
+    },
+    ...
+  ],
+  "secondCourses": [
+    {
+      "name": "Salad",
+      "description": "Tomatoes, lettice and bacon.",
+      "image": "https://www.google.es/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwjknLawgs3TAhULVxQKHfbcAvMQjRwIBw&url=http%3A%2F%2Fjetspizza.com%2Fmenu%2Fsalads&psig=AFQjCNE447cnpJfGRhiwiN5Hys_XLrgpmQ&ust=1493670502918204"
+    },
+    ...
+  ],
+  "desserts": [
+   {
+      "name": "Salad",
+      "description": "Tomatoes, lettice and bacon."
+      "image": "https://www.google.es/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwjknLawgs3TAhULVxQKHfbcAvMQjRwIBw&url=http%3A%2F%2Fjetspizza.com%2Fmenu%2Fsalads&psig=AFQjCNE447cnpJfGRhiwiN5Hys_XLrgpmQ&ust=1493670502918204"
+    },
+    ...
+  ]
+}
+```
+
+## Menu choice
```

Comment:
If you add this method here, why you don't  add in book the method of lending a book. I think that is necessary create in book this method.